src/instructlab/model/evaluate.py

```diff
@@ -100,6 +100,10 @@ def get_evaluator(
                     fg="red",
                 )
                 raise click.exceptions.Exit(1)
+            click.secho(
+                f"Using local model found at '{model}' for 'model'",
```

Comment:
My thought was that it might be good if we do similar statements for things like "judge-model" and "base-model"

Reply:
I could include statements like that as part of this PR? Wanted to limit the scope but maybe it's worth doing to demonstrate what I'm talking about

Reply:
OK if you mean you'd like to refer to the CLI option, then maybe consider saying `for --model`? So that it's more clear you are talking about CLI args.

Reply:
Makes sense to me